blueprints/node-webkit/files/nw-runner.js

```diff
@@ -0,0 +1,112 @@
+var exec = require('child_process').exec;
+var NW_VERSION = '0.12.0';
+var os = require('os');
+var fs = require('fs');
+
+runNw('./node_modules/nw/bin/nw');
+
+function runNw(nwPath){
+    var nw;
+    if(os.platform() === 'linux'){
+        // TODO - What happens to folks running a linux desktop?
+        nw = exec('./travis_runner.sh ' + nwPath + ' ./dist/tests');
+    }
+    else{
+        nw = exec(nwPath + ' ./dist/tests');    
+    }
+    
+    nw.stdout.on('data', function (data) {
+        process.stdout.write(data);
+    });
+    var onError = false;
+
+    // Cleanup nw.js output to be TAP (test anything protocol) compliant
+    nw.stderr.on('data', function (data) {
+        if(data.indexOf('DEBUG:') === -1 &&
```

Comment:
This seems a bit brittle. Since you have control over what QUnit outputs in `qunit-logger.js`, would it be better to wrap the lines that you need in something that you can easily search for here, e.g. "[qunit-logger] ok 1 ..."?